panoramix/models.py

```diff
@@ -862,6 +884,54 @@ def generate_metrics(self):
             col.generate_metrics()
 
     @classmethod
+    def sync_to_db2(cls, datasource_config, cluster):
```

Comment:
It would be nice to refactor this into "contrib" dir and some how include it dynamically into Panoramix.